app/support/reservations/validations.rb

```diff
@@ -23,12 +23,12 @@ module Reservations::Validations
     validate :duration_is_interval
 
     validates :category,
-              inclusion: { in: -> (r) { r.class::CATEGORIES }, allow_blank: false },
+              inclusion: { in: -> (r) { r.class::CATEGORIES }, allow_blank: true },
               if: -> (r) { r.class.const_defined?(:CATEGORIES) }
 
     validates :category,
               absence: true,
-              unless: -> (r) { r.class.const_defined?(:CATEGORIES) }
+              unless: -> (r) { r.class.const_defined?(:CATEGORIES) || r.admin? }
```

Comment:
I think moving the `category` inclusions to the subclasses is a good idea. I kind of like enforcing no category for plain `Reservation` but if removing it makes these validations less complicated then let's take it out.